fonts/README.md

```diff
@@ -8,7 +8,9 @@ This directory contains the fonts used internally by MuseScore and the sources u
 
 * **bravura** The [_Bravura_](https://github.com/steinbergmedia/bravura/) fonts and the documentation for them. These fonts are **not** maintained by the MuseScore community, please **DO NOT EDIT** these fonts. 
 * **campania** The [_Campania_](https://github.com/MarcSabatella/Campania) font for Roman numeral analysis.  This font is not maintained by the MuseScore project, but it is open source and contributions from the MuseScore community are encouraged at the main site for the font.
+* **edwin** The [_Edwin_] text font family. This font is maintained by the MuseScore project.
```

Comment:
Shouldn't the [ ] be left out, because there is no hyperlink?

Reply:
Yes, thanks